test/integrations/destinations/intercom/router/data.ts

```diff
@@ -0,0 +1,302 @@
+export const data = [
+  {
```

Comment:
Same as above can we add a negative test cases here

Reply:
all sorts of negative tests already added in processor

Reply:
A negative in douter will confirm that router processing is also working fine

Reply:
Addressed